crates/biome_js_analyze/src/lint/style/use_import_type.rs

```diff
@@ -109,6 +104,52 @@ declare_lint_rule! {
     /// import { B } from "./mod.js" with {};
     /// export type { B };
     /// ```
+    ///
+    /// ## Options
+    ///
+    /// This rule respects the [`jsxRuntime`](https://biomejs.dev/reference/configuration/#javascriptjsxruntime)
+    /// setting and will make an exception for React globals if it is set to
+    /// `"reactClassic"`.
+    ///
+    /// ### `style`
+    ///
+    /// The `style` option allows enforcing a style for importing types.
+    /// The option supports three values:
+    ///
+    /// - `inlineType`: always use `import { type T }` instead of `import type { T }`
+    /// - `separatedType`: always use `import type { T }` instead of `import { type T }`
```

Comment:
Bikeshedding: How about `outerType`?

Reply:
I personally prefer `separatedType` because this is aligned with the TS Eslint rule.